aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/transformations/DataTableTypeTransformations.java

```diff
@@ -0,0 +1,19 @@
+package aquality.selenium.template.cucumber.transformations;
+
+import aquality.selenium.template.models.ContactUsInfo;
+import io.cucumber.java.DataTableType;
+
+import java.util.Map;
+
+public class DataTableTypeTransformations {
+
+    @DataTableType
+    public ContactUsInfo getContactUsInfo(Map<String, String> entry) {
+        ContactUsInfo contactUsInfoModel = new ContactUsInfo();
```

Comment:
Just wonder whether this could be autogenerated for all models, or should be defined for each model manually

Reply:
As far as I know it is not possible to do it from the box. We have to define transformation for every model.